docs/source/usage/generating_synthetic_data.rst

```diff
@@ -0,0 +1,32 @@
+Generating Synthetic Data
+=========================
+
+MetaSynth can **generate synthetic data** from any :obj:`MetaDataset <metasynth.dataset.MetaDataset>`.
```

Comment:
MetaDataset can now be updated to MetaFrame.

Reply:
I made an issue for that, I'll go over the renaming after this has been merged into main.